frontend/www/js/logout.ts

```diff
@@ -23,21 +15,30 @@ omegaup.OmegaUp.on('ready', function() {
     // Promise objects, therefore they don't have an .else() or .finally().
     // That trips up the linter.
     gapi.auth2.init({})['then'](
-      function(auth) {
+      function(auth: any) {
         auth.signOut()['then'](
           function() {
             redirect();
           },
-          function(error) {
+          function(error: any) {
             console.error(error);
             redirect();
           },
         );
       },
-      function(error) {
+      function(error: any) {
```

Comment:
El tipo que regresa es una Promesa, y hay que usar arrow function, pudiera quedar así:
```suggestion
      (error: Promise<String>) => {
```

Reply:
listo